update_kb_redis.py

```diff
@@ -569,9 +569,9 @@ def process_keyboard(keyboard, usb_list, kb_list, kb_entries):
 
     # Setup platform specific keys
     mcu = rules_mk.get('MCU')
-    if mcu in ARM_PROCESSORS:
+    if mcu in ARM_PROCESSORS or mcu.startswith(STM32_PROCESSORS):
```

Comment:
Why not
```suggestion
    if mcu in ARM_PROCESSORS or mcu.startswith('STM32'):
```
so this can closely follow the CLI's [constant.py](https://github.com/qmk/qmk_firmware/blob/master/lib/python/qmk/constants.py) file?

Reply:
Then `STM32_PROCESSORS` isn't used at all... 🤔

Reply:
Exactly.

Reply:
But what if we need a nice list of all the STM32 MCUs currently supported?

Reply:
`[mcu for mcu in ARM_PROCESSORS if mcu.startswith('STM32')]`

Do you have a use-case for this in mind?

Reply:
Not particularly. Now that I think about it, since I'm splitting up the AVR MCUs here into LUFA and V-USB, the ARM ones should really just be renamed to `CHIBIOS_PROCESSORS`...and perhaps we could get away with `mcu.startswith(CHIBIOS_PROCESSORS)`.